toolkit/tools/imagecustomizerapi/injectfilesconfig.go

```diff
@@ -7,11 +7,13 @@ import (
 	"fmt"
 )
 
-type InjectFilesYaml struct {
+// InjectFilesConfig defines the list of files to be injected into partitions.
+type InjectFilesConfig struct {
 	InjectFiles []InjectArtifactMetadata `yaml:"injectFiles" json:"injectFiles,omitempty"`
```

Comment:
We should probably mark this API as a preview feature. So, lets add a `PreviewFeatures` field here. And name the feature something like `inject-files`.

Reply:
Nice advice, thanks.